{"schemaVersion":{"patch":0,"major":0,"minor":3},"variants":[{"paths":["\/documentation\/sotocore\/awsresponseerror\/context"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","abstract":[{"text":"additional context information related to the error","type":"text"}],"identifier":{"url":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError\/context","interfaceLanguage":"swift"},"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["Linux"],"tokens":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"context","kind":"identifier"},{"text":": ","kind":"text"},{"text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV","kind":"typeIdentifier","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext"},{"text":"?","kind":"text"}]}],"kind":"declarations"}],"sections":[],"hierarchy":{"paths":[["doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError"]]},"metadata":{"fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"context","kind":"identifier"},{"text":": ","kind":"text"},{"text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"title":"context","symbolKind":"property","modules":[{"name":"SotoCore"}],"externalID":"s:8SotoCore16AWSResponseErrorV7contextAA15AWSErrorContextVSgvp","role":"symbol","roleHeading":"Instance Property"},"references":{"doc://codes.soto.soto-core/documentation/SotoCore/AWSResponseError":{"fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AWSResponseError","kind":"identifier"}],"type":"topic","url":"\/documentation\/sotocore\/awsresponseerror","role":"symbol","abstract":[{"type":"text","text":"Response Error type returned by Soto if the error code is unrecognised"}],"kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError","title":"AWSResponseError","navigatorTitle":[{"text":"AWSResponseError","kind":"identifier"}]},"doc://codes.soto.soto-core/documentation/SotoCore":{"type":"topic","url":"\/documentation\/sotocore","role":"collection","abstract":[{"text":"The core framework for Soto, a Swift SDK for AWS","type":"text"}],"kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","title":"SotoCore"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSResponseError/context":{"fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"context","kind":"identifier"},{"text":": ","kind":"text"},{"text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"type":"topic","abstract":[{"text":"additional context information related to the error","type":"text"}],"url":"\/documentation\/sotocore\/awsresponseerror\/context","kind":"symbol","role":"symbol","title":"context","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError\/context"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSErrorContext":{"fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AWSErrorContext","kind":"identifier"}],"type":"topic","url":"\/documentation\/sotocore\/awserrorcontext","role":"symbol","abstract":[{"text":"Additional information about error","type":"text"}],"kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","title":"AWSErrorContext","navigatorTitle":[{"text":"AWSErrorContext","kind":"identifier"}]}}}